lib/gc/Transforms/GPU/LinalgToXeGPU.cpp

```diff
@@ -733,16 +749,19 @@ loadNdDescTiles(PatternRewriter &rewriter, Location loc, ValueRange loadTiles,
   VectorType vecLoadType =
       VectorType::get(tileType.getShape(), tileType.getElementType());
   IntegerAttr vnniAxisAttr = nullptr;
+  mlir::UnitAttr packedAttr = nullptr;
   if (vnniConf) {
     vnniAxisAttr = IntegerAttr::get(rewriter.getI64Type(), vnniConf->vnniAxis);
     vecLoadType = getVnniVector(tileType.getShape(), tileType.getElementType(),
                                 *vnniConf);
+    packedAttr = mlir::UnitAttr::get(rewriter.getContext());
   }
-
+  IntegerAttr transpose_bit = nullptr;
   SmallVector<Value> loadVec;
   for (auto tile : loadTiles) {
+
     auto loadOp = rewriter.create<xegpu::LoadNdOp>(
-        loc, vecLoadType, tile, vnniAxisAttr, transpose,
+        loc, vecLoadType, tile, packedAttr, transpose, transpose_bit,
```

Comment:
1. [`vnniAxis` -> `packedAttr`](https://github.com/intel/mlir-extensions/pull/793): instead of a vnni axis (0, 1) specify "packed" attribute that's equivalent of `vnni_axis=0`
2.  [`transpose_bit`](https://github.com/intel/mlir-extensions/blob/main/build_tools/patches/0008-amend-xegpu-transpose_bit_width-and-qualified-type-f.patch): allows to transpose data while loading. Isn't used by this lowering pass